README.md

```diff
@@ -151,7 +151,8 @@ $ sudo apt-get install   \
       libdbus-glib-1-dev \
       build-essential    \
       libelf-dev \
-      libseccomp-dev
+      libseccomp-dev \
+      libclang-dev
```

Comment:
Had to add this, otherwise it wouldn't compile on my system. @utam0k can you check?

Reply:
sorry but I don't have my laptop now. I wonder why the CI passde without it.